src/components/BigButton.js

```diff
@@ -4,6 +4,10 @@ import { theme } from '../infrastructure/index';
 
 const styles = StyleSheet.create({
   button: {
+    shadowColor: 'rgba(0,0,0, .4)', // IOS
+    shadowOffset: { height: 1, width: 1 }, // IOS
+    shadowOpacity: 1, // IOS
+    shadowRadius: 1, //IOS
```

Comment:
Right now, for our app, iOS is good. But to keep it general, you could do something like this. It's non-blocking now. Just a way to do this.
`
import { Platform } from 'react-native';
shadowColor: Platform.OS  === 'ios' ? rgba(0,0,0,.4) : rgba values for android
`

Reply:
Ok cool, good to know. the "elevation" property is android equivalent of that code, and all that is copy and pasted :)